Practiece12/Task5/Flowers.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+
+namespace Task5
+{
+    public class Flowers
+    {
+        public Flowers(double price)
+        {
+            Price = price;
+        }
+
+        private double _price;
+        public double Price
+        {
+            get => _price;
+            set
+            {
+                if (value > 0)
+                {
+                    _price = value;
+                }
+            }
+        }
+
+        public static double CostOfBouquet(List<Flowers> bouquet)
+        {
+            double costOfBouquet = 0;
```

Comment:
why did you put with method here? is it really related to the flower hierarchy?

Reply:
already fixed)